### PR TITLE
[AMORO-2029]: Using ArcticTableOperation in mixed-iceberg

### LINKED ISSUE #2029: [Bug]: Dashboard can't show files page with kerberos and mixed-iceberg format in external catalog
### What happened?

图片

图片

### Affects Versions

master

### What engines are you seeing the problem on?

Core

### How to reproduce

1. Create External Catalog with Mixed-Iceberg format
2. Catalog authed by Kerberos.
3. Show files will failed.

### Relevant log output

_No response_

### Anything else

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by baiyangtx**
## Why are the changes needed?
 

Close #2029.

## Brief change log

- Wrapper tables with ArcticTableOperations when load/create table.

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [X] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Review by wangtaohz — COMMENTED** [review]
Thanks for your contribution. I left some comments, PTAL.

File: core/src/main/java/com/netease/arctic/mixed/MixedTables.java [hunk 1]
```diff
@@ -100,13 +101,20 @@ public ArcticTable loadTable(Table base, com.netease.arctic.table.TableIdentifie
     ArcticFileIO io = ArcticFileIOs.buildAdaptIcebergFileIO(this.tableMetaStore, base.io());
     PrimaryKeySpec keySpec = getPrimaryKeySpec(base);
     if (!keySpec.primaryKeyExisted()) {
-      return new BasicUnkeyedTable(tableIdentifier, base, io, catalogMeta.getCatalogProperties());
+      return new BasicUnkeyedTable(tableIdentifier,
+          CatalogUtil.useArcticTableOperations(
```
[the remark this answers is not in the document]
Comment (baiyangtx) [reply core/src/main/java/com/netease/arctic/mixed/MixedTables.java]: fixed

**Review by wangtaohz — APPROVED** [review]
LGTM.
